tests/unit/test_tag_google_analytics_gtag.py

```diff
@@ -59,6 +59,21 @@ def test_identify(self):
         r = GoogleAnalyticsGTagNode().render(Context({'user': User(username='test')}))
         assert "gtag('set', {'user_id': 'test'});" in r
 
+    def test_identity_context(self):
+        """ Test when identity is set in the context """
```

Comment:
Don't use spaces at the beginning and the end of the docsstring.

Don't mention the obvious ("Test when"), get straight to the point instead, e.g. 

```
        """Identity must be set in the context when ..."""
```

Reply:
OK. The docstring is now too long to be one-liner, so it has to be split anyway.